### ISSUE TITLE
No way to resend account confirmation email

### ISSUE THREAD

**Issue by BK-TN** [gamejolt/issue-tracker#706]
After creating a new account, Game Jolt sends a confirmation email to your adress that you use in order to activate it.
However once this mail is sent there is no way to resend it in case it didn't arrive for whatever reason.
There's also no way to log into the account or to use the same email or username again..  You'll have to wait a full week for the site to cancel the account creation process.

**Comment by YLivay** [comment]
Agreed, we should be able to re-send. For now they can contact contact@gamejolt.com for manual resending.
